junit-jupiter-engine/src/main/java/org/junit/jupiter/engine/descriptor/TestFactoryTestDescriptor.java

```diff
@@ -88,8 +88,12 @@ protected void invokeTestMethod(JupiterEngineExecutionContext context) {
 
 			try {
```

Comment:
ah yes ofcourse, for some reason my brain failed to associate the close in stream method with autoclosable, thanks!